yasa/tests/test_hypno.py

```diff
@@ -126,3 +128,32 @@ def test_periods(self):
         assert_frame_equal(
             hfp(np.array(x).astype(str), sf_hypno=1 / 60, threshold="0min"), expected, **kwargs
         )
+
+    def test_simulation(self):
+        """Test hypnogram simulations."""
+        hypno = simulate_hypno(tib=360, sf=1 / 30)
+        assert hypno.size <= 360 * 60 * 1 / 30
+        assert np.unique(hypno).size > 1
+        assert np.array_equal(simulate_hypno(tib=4, seed=7), np.array([0, 0, 0, 0, 1, 1, 1, 2]))
+        assert np.unique(simulate_hypno(tib=500, n_stages=2)).size == 2
+
+        # Passing in probabilities
+        trans_df = pd.DataFrame(np.full((5, 5), 0.2))
```

Comment:
Could you also add a test with `trans_df = pd.DataFrame(np.eye(5, 5))`? The resulting hypnogram should be all wake, right?